presto-main/src/main/java/com/facebook/presto/execution/CallTask.java

```diff
@@ -131,7 +131,7 @@ else if (i < procedure.getArguments().size()) {
 
         // get argument values
         Object[] values = new Object[procedure.getArguments().size()];
-        Map<NodeRef<Parameter>, Expression> parameterLookup = parameterExtractor(call, parameters);
+        Map<NodeRef<Parameter>, Expression> parameterLookup = parameterExtractor(call, parameters).getFirstRowOfParametersIfExists();
```

Comment:
Should we add a method on `MultiLineParameters` that doesn't just returns the empty map if it doesn't exist, but throws with an exception to fail fast and alert on the bug?